### PR TITLE
Feature/filter join games

### PR DESCRIPTION
**Pull request by ArnaudPoletto**
Adds filter related to the name of games to the game list.

### CODE REVIEW

File: app/src/androidTest/java/sdp/moneyrun/MenuActivityTest.java [hunk 1]
```diff
@@ -132,6 +125,57 @@ public void joinGamePopupIsDisplayed() {
         }
     }
 
+    @Test
+    public void filterWorks(){
```
Comment (Tesa27) [inline app/src/androidTest/java/sdp/moneyrun/MenuActivityTest.java:129]: I think you should add one more test hat tests a case where the filter does not work

[the remark this answers is not in the document]
Comment (ArnaudPoletto) [reply app/src/androidTest/java/sdp/moneyrun/MenuActivityTest.java:132]: done

Comment (ArnaudPoletto) [reply app/src/androidTest/java/sdp/moneyrun/MenuActivityTest.java:132]: done

[the remark this answers is not in the document]
Comment (ArnaudPoletto) [reply app/src/androidTest/java/sdp/moneyrun/MenuActivityTest.java:132]: Indeed, I wasn't clearly testing the filtering property

**Review by gauthierboeshertz — APPROVED** [review]

**Review by Tesa27 — APPROVED** [review]
All good now!

**Pull request merged**